src/firmware/boards/robot_stm32h7/io/allegro_a3931_motor_driver.h

```diff
@@ -65,3 +61,10 @@ void io_allegro_a3931_motor_driver_setDirection(
  */
 void io_allegro_a3931_motor_setPwmPercentage(AllegroA3931MotorDriver_t* motor_driver,
                                              float pwm_percentage);
+
+/**
+ * Disable the motor driver, allows the motors to spin freely
+ *
+ * @param motor_driver
+ */
+void io_allegro_a3931_motor_disable(AllegroA3931MotorDriver_t* motor_driver);
```

Comment:
Why is there no corresponding `enable` method?

Reply:
it just gets enabled when you set the pwm

Reply:
We should probably specify this. I can imagine someone writing code like:

```
if (some_condition) {
    disable(my_driver);
}

// Since we disable the driver above, this has no effect if 
// some_condition is true
set_pwm(my_driver, 100);
```
(I know the comment is wrong, I'm trying to convey how someone could think this function works).

Reply:
yeah thats true, added a comment